backend/src/v1/services/external-consumer-service.ts

```diff
@@ -188,5 +188,48 @@ const externalConsumerService = {
       ],
     };
   },
+  async deleteReports(bceid_business_guid: string) {
```

Comment:
delete reports should be moved to `report-service.ts` and SHOULD NOT USE `REPLICA` and it should accept parameters based on which it can delete the reports.